travis-cabal.sh

```diff
@@ -4,6 +4,12 @@
 set -euo pipefail
 set -x
 
+echo Installing hpack
+curl -OL https://github.com/sol/hpack/releases/download/0.28.2/hpack_linux.gz
+gunzip hpack_linux.gz
+chmod +x hpack_linux
+mv hpack_linux $HOME/.local/bin/hpack
```

Comment:
Maybe not assume that cwd is writable?
curl -L https://github.com/sol/hpack/releases/download/0.28.2/hpack_linux.gz | gunzip > $HOME/.local/bin/hpack
chmod +x $HOME/.local/bin/hpack